src/js/actions/layers.js

```diff
@@ -1642,8 +1647,8 @@ define(function (require, exports) {
      * though possibly out of order w.r.t. Photoshop's model.
      *
      * @param {Document=} document Document for which layers should be reordered, if undefined, use current document
-     * @param {boolean=} suppressHistory if truthy, dispatch a non-history-changing event.
-     * @param {boolean=} amendHistory if truthy, update the current state (requires suppressHistory)
+     * @param {boolean=} suppressHistory Optional. If true, emit an event that does NOT add a new history state
```

Comment:
reviewer note: I just changed this JS Doc to have the same wording as resetBounds because the two both use the same basic pattern.